static/js/components/dashboard/FinalExamCard.js

```diff
@@ -267,28 +267,55 @@ export default class FinalExamCard extends React.Component<void, Props, void> {
       },
       showPearsonTOSDialog
     } = this.props
-
-    switch (program.pearson_exam_status) {
-    case PEARSON_PROFILE_ABSENT:
-      return absentCard()
-    case PEARSON_PROFILE_SUCCESS:
-      return successCard(profile, navigateToProfile)
-    case PEARSON_PROFILE_IN_PROGRESS:
-      return pendingCard()
-    case PEARSON_PROFILE_INVALID:
-      return invalidCard(navigateToProfile)
-    case PEARSON_PROFILE_SCHEDULABLE:
-      return schedulableCard(
-        profile,
-        program,
-        navigateToProfile,
-        pearson,
-        showPearsonTOSDialog,
-        pearsonTOSDialogVisible,
-        submitPearsonSSO
+    if (!SETTINGS.FEATURES.ENABLE_EDX_EXAMS) {
+      switch (program.pearson_exam_status) {
```

Comment:
Is this part deprecated now?

Reply:
This value is still used to determine if the state of the exam card. It needs to be renamed, but I guess I thought to postpone it. Do you think it is better to rename it in this PR?

Reply:
No, I'm fine leaving it for now